css-layout-classes/index.html

```diff
@@ -0,0 +1,78 @@
+  <head>
+    <title>CSS Layout Classes</title>
+    <link href="styles.css" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose your character!</h1>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="link">
+        </div>
+        <div class="column-one-third">
+          <h1>Super Smash Bros. Ultimate</h1>
+          <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players,
+            with some critics calling it the best installment in the series.  It received praise for its
+            large amount of content and fine-tuning of existing Smash gameplay elements, although
+            its online mode was widely criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png" alt="yoshi">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png" alt="fox">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png" alt="pikachu">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png" alt="ness">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <h1>Donkey Kong</h1>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a gorilla
+            called Donkey Kong, concieved by Shigeru Miyamoto in 1981.  The franchise consists mainly
+            of two game genres, but also includes additional spin-off titles of various genres
+          </p>
+          <p>
+            The games of the first genre are mostly single-screen platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
+        <div class="column-one-half">
+          <img src="images/dk.png" alt="donkey-kong">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <img src="images/mario.png" alt="mario">
+        </div>
+        <div class="column-one-half">
+          <h1>Mario</h1>
+          <p>
+            Mario is a fictional character in the Mario video game franchise,
+              owned by Nintendo and created by Japanese video game designer Shigeru
+              Miyamoto.  Acting as the company's mascot, as well as being the eponymous
+              protagonist of the series, Mario has appeared in over 200 video games since his creation.
+          </p>
+          <p>
+              Depicted as a short, pudgy, Italian plumber who resides in the Mushroom Kingdom,
+              his adventures generally center upon rescuing Princess Peach from the Koopa villain
+              Bowser.
+            </p>
+        </div>
```

Comment:
`column-one-half` doesn;t really match the intended size of these columns 🤔 

You should probably have a class named something like `column-one-third` and `column-two-thirds`